src/main/java/fish/payara/jcache/api/NamedCache.java

```diff
@@ -36,7 +36,7 @@
  * only if the new code is made subject to such option by the copyright
  * holder.
  */
-package fish.payara.cdi.jsr107.impl;
```

Comment:
We'll need to mention the change in the package name in the release notes, in case someone already uses this annotation.